index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1 id="title"> hola </h1>
+
+    <p id="description">Hello World!, we are software developers!</p>
+
```

Comment:
I suggest you use semantic tags like a `<header>, <main>, <section>, etc `.
Below the `body` can use `main` semantic tag. 😮